cmd/zpool/zpool_main.c

```diff
@@ -2163,19 +2173,84 @@ show_import(nvlist_t *config)
 	}
 }
 
+typedef long long unsigned int llu;
+
+static void
+extract_timestamp_and_txg(nvlist_t *config, uint64_t *txg,
+    uint64_t *timestamp)
+{
+	verify(nvlist_lookup_uint64(config, ZPOOL_CONFIG_TIMESTAMP,
+	    timestamp) == 0);
+	verify(nvlist_lookup_uint64(config, ZPOOL_CONFIG_FOUND_TXG,
+	    txg) == 0);
+}
+
+static int
+check_pool_for_activity(libzfs_handle_t *hdl, nvlist_t *config)
+{
+	nvlist_t *nvl = NULL;
+	uint64_t original_txg, original_timestamp;
+	int rc = 0;
+	char *poolname;
+	int seconds_between_checks = 1;
+	int times_to_check = 10;
```

Comment:
@richardelling ,

Thanks for your feedback. 

The seconds-between-checks is just to save the user time - under many normal conditions, at least one uberblock will change either because of a sync or because the mmp thread updated one.  So checking frequently lets the import fail faster.

Note that any uberblock update on any vdev will do.  

You make a good point that the I/O to get the updated config might take a long time.  Perhaps I should decide when to exit the loop and move on with the import based on the amount of time that has passed, not the number of iterations through the loop.

There is a problem if 

(seconds_between_checks*times_to_check) < (delay between uberblock updates)

and from a safety perspective, it seems to me that waiting longer is safe, but waiting less time is not.  So I'm OK with a parameter as long as I enforce a minimum amount of time.  That being said, I know @behlendorf  has concerns about settling on a user interface that we regret later.  Let's get his thoughts as well.

What do you think a reasonable delay is?